vision/object_detection_segmentation/retinanet/README.md

```diff
@@ -0,0 +1,98 @@
+# RetinanNet
+
+## Description
+[RetinaNet](https://github.com/NVIDIA/retinanet-examples) is a single-stage object detection model.
+
+## Model
+
+|Model        |Download  | Download (with sample test data)|ONNX version|Opset version|Accuracy |
+|-------------|:--------------|:--------------|:--------------|:--------------|:--------------|
+|RetinaNet (ResNet101 backbone)|    [228.4 MB](model/retinanet.onnx)   | [143.7 MB](model/retinanet.tar.gz)    |  1.6.0  |9| mAP 0.376      |
+
+## Inference
+A sample script for ONNX model conversion and ONNXRuntime inference can be found [here](dependencies/retinanet-export.py).
+
+### Input
+The model expects mini-batches of 3-channel input images of shape (N x 3 x H x W), where N is batch size.
+
+### Preprocessing
+The images have to be loaded in to a range of [0, 1] and then normalized using mean = [0.485, 0.456, 0.406] and std = [0.229, 0.224, 0.225]. The transformation should preferably happen at preprocessing.
+
+The following code shows how to preprocess a NCHW tensor:
+
+```python
+import numpy as np
+
+def preprocess(img_data):
+    mean_vec = np.array([0.485, 0.456, 0.406])
+    stddev_vec = np.array([0.229, 0.224, 0.225])
+    norm_img_data = np.zeros(img_data.shape).astype('float32')
+    for i in range(img_data.shape[0]):  
+         # for each pixel in each channel, divide the value by 255 to get value between [0, 1] and then normalize
+        norm_img_data[i,:,:] = (img_data[i,:,:]/255 - mean_vec[i]) / stddev_vec[i]
+    return norm_img_data
+```
+
+
+### Output
+
+Model has 2 outputs:
+
+Classification heads: List of classification heads.
```

Comment:
Can you indicate dimensions of these model outputs?

Reply:
I updated this with more details. Model output dimensions depend on model params, and output size depends on Conv layer sizes.
If it's helpful, I can mention the dimension values for this pretrained model I've used as an example.

Reply:
Yes, I think the dimension values in the model you submitted would be helpful to include. I see the following: 

![image](https://user-images.githubusercontent.com/10929090/83581269-33208980-a4f3-11ea-9925-74179c7de037.png)